_posts/2022-11-01-ndk_advanced_custom_fields-CVE-2022-40840.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title: "[CVE-2022-40840][CVE-2022-40841] ndk design NdkAdvancedCustomizationFields 3.5.0 is vulnerable to Cross Site Scripting (XSS) via createPdf.php"
+categories: module
+author:
+- Friends-Of-Presta.org
+meta: "CVE,PrestaShop,ndkadvancedcustomizationfields"
+severity: "medium (6.8)"
+---
+
+Ndk design NdkAdvancedCustomizationFields 3.5.0 is vulnerable to Reflected Cross Site Scripting (XSS-T1 of category 1) via createPdf.php and showPreview.php.
```

Comment:
Same suggestion as the title : "Ndk advanced custom fields" module from ndkdesign ...